smartsim/_core/control/controller.py

```diff
@@ -572,7 +582,11 @@ def _launch_step(
         if completed_job is None and (
             entity.name not in self._jobs.jobs and entity.name not in self._jobs.db_jobs
         ):
-            self.symlink_output_files(job_step, entity)
+            # make sure it's not an _AnonymousBatchJob
```

Comment:
From a little further back, it seems that `_launch_step` is now completely in the business of doing symlinks. Can you consider extracting a method for the expanded symlinking needs instead of making `_launch_step` do it all?

Reply:
What do you think about adding the if check to the `symlink_output_files` method?

Reply:
just kidding that doesn't work.

Reply:
`_launch_step` has retired from the business of doing symlinks